src/app/space-wizard/common/logger.ts

```diff
@@ -8,7 +8,7 @@ export interface ILogEntry {
   inner?: any;
 }
 export interface ILoggerDelegate {
-  (options: string | ILogEntry): void;
+  (options: string | ILogEntry,...args): void;
```

Comment:
an interface in typcript can also be used to defined function signatures .... the  ...args makes it easier to handle variable argument list ... whihc i added to make the logging collasable